Source/WebCore/Modules/fetch/FetchBodyOwner.cpp

```diff
@@ -138,47 +137,42 @@ void FetchBodyOwner::blob(Ref<DeferredPromise>&& promise)
         return;
     }
     m_isDisturbed = true;
-    m_body->blob(*this, WTFMove(promise), m_contentType);
+    m_body->blob(*this, WTFMove(promise));
 }
 
 void FetchBodyOwner::cloneBody(FetchBodyOwner& owner)
 {
     m_loadingError = owner.m_loadingError;
-
-    m_contentType = owner.m_contentType;
+    m_headers = FetchHeaders::create(owner.m_headers.get());
     if (owner.isBodyNull())
         return;
     m_body = owner.m_body->clone();
 }
 
 ExceptionOr<void> FetchBodyOwner::extractBody(FetchBody::Init&& value)
 {
-    auto result = FetchBody::extract(WTFMove(value), m_contentType);
+    auto currentContentType = contentType();
+    auto wasEmptyContentType = currentContentType.isNull();
```

Comment:
wasEmptyContentType is not a great name, the isNull check means there was no content-type header, it does not mean there was a content-type header with an empty value.

Reply:
you're right. please let me rename it to `contentTypeNotExist` 🙏